doc/architecture/feature_data.adoc

```diff
@@ -0,0 +1,5 @@
+= Feature data
+
+Feature data contains detected features in the reference frame of a sensor.
+Feature data is generated from ground-truth data.
+It serves, for example, as input for sensor models simulating object detection or feature fusion models.
```

Comment:
Feature data needs more description, I feel that it is still vague.

Reply:
I agree that the topic can be expanded. Do you have any suggestions? What are you missing?